rdr_service/config.py

```diff
@@ -232,6 +232,8 @@
 RDR_SLACK_WEBHOOKS = "rdr_slack_webhooks"
 RDR_VALIDATION_WEBHOOK = "rdr_validation_webhook"
 
+NPH_SLACK_WEBHOOKS = "nph_slack_webhooks"
```

Comment:
Sorry not seeing this being utilized in this PR, I think this ticket should just be the backend/data for incidents, so we can remove this for now.

Reply:
Good Catch, I forgot to add the module that uses it

Reply:
This `services/ancillary_studies/nph_incident.py` module uses it.